components/picker/index.css

```diff
@@ -265,7 +265,7 @@
 			border-color: var(--highcontrast-picker-focus-indicator-color, var(--mod-picker-focus-indicator-color, var(--spectrum-picker-focus-indicator-color)));
 		}
 
-		&.is-placeholder {
+		.is-placeholder {
```

Comment:
Let me know if I'm missing anything by removing `&` here. I felt like I was fighting the specificity when further down in this file, there's another `color` style declaration.

Reply:
This looks like the right correction to the selector, since `.is-placeholder` doesn't live on the root class according to the template (it's on the nested span with the `spectrum-Picker-label` class). 👍

Reply:
My only concern is in having an `is-` prefixed class unattached to a specific element. Should we add `.spectrum-Picker-label` in front of it to ensure we don't end up with an unexpected nested class getting targeted?